src/interface/lazy.jl

```diff
@@ -11,7 +11,9 @@ end
 LazyTensor{T}(data, extrude::NTuple{N, Bool}) where {T, N} = LazyTensor{T, N}(data, extrude)
 
 Base.ndims(::Type{LazyTensor{T, N}}) where {T, N} = N
+Base.ndims(::LazyTensor{T, N}) where {T, N} = N
```

Comment:
This is usually handled by the `Base.AbstractArray` class, but sometimes making things subtype `AbstractArray` is more trouble than it's worth. The fallback is also sometimes written as `Base.ndims(tns::LazyTensor) = ndims(typeof(tns))`

I'm fine with this code as is just thought I'd elaborate on what's going on. You can read more about this here:

https://docs.julialang.org/en/v1/manual/interfaces/#man-interface-array

Reply:
Sure! I added a fallback-style function.